pkg/notebook/reconcile.go

```diff
@@ -65,7 +67,11 @@ func (r Reconciler) Reconcile() error {
 }
 
 func (r Reconciler) reconcileDeployment() error {
-	desired := r.gen.DesiredDeploymentWithoutOwner()
+	desired, err_ := r.gen.DesiredDeploymentWithoutOwner()
+	if err_ != nil {
+		r.log.Error(err_, "Failed to get desired deployment")
```

Comment:
No need to log it since we will print the error outside.

Reply:
Fixed